readme.md

```diff
@@ -117,10 +117,18 @@ Now, running `node example.js` yields:
 This package exports no identifiers.
 The default export is `rehypeSlug`.
 
-### `unified().use(rehypeSlug)`
+### `unified().use(rehypeSlug[, options])`
 
 Add `id`s to headings.
-There are no options.
+
+##### `options`
+
+Configuration (optional).
+
+###### `options.prefix`
+
+String to insert in front of the `id`s (optional).
+By default, an empty string (`''`) is used.
```

Comment:
```suggestion
Prefix to add in front of `id`s (`string`, default: `''`).
```

I think this is better. I like to use the word prefix, instead of a type, which is more jargon-y and less natural language. Same with `add` instead of `insert`.

The type can go in brackets. As there’s a default here, showing that implies that it is optional.

Maybe reuse it in the JSDoc description as well (except for the ``((`string`, default: `''`))`` part)